base.py

```diff
@@ -38,7 +36,7 @@ def transform(self, X: Union[pd.DataFrame, np.ndarray]):
                 X[i : i + batch_size] for i in range(0, len(X), batch_size)
             )
 
-            with joblib.parallel_backend(n_jobs=self.n_jobs):
+            with joblib.parallel_backend(backend="loky",n_jobs=self.n_jobs):
                 results = Parallel()(
```

Comment:
Tutaj można podać `n_jobs` wprost do `Parallel`, i nie podawać jawnie backendu. W `Parallel` domyślny backend to i tak `loky`

Reply:
Tutaj jeszcze mamy parallel_backend który daje błąd jak nie przekażemy backend-u. Zmiana na Parallel będzie zadaniem w kolejnych sprintach. W poprzednim pr-rze o tym wspominaliśmy